#159459817 Update README.md Instructions

davenjeru opened the pull request:
**What does this PR do?**
Updates the README instructions.

**Description of tasks to be completed.**

- Update the request data requirements such that information does not need to be wrapped in a redundant keyword like "user", "article" etc.

**How can this be manually tested?**
Take a look at this README.md [here](https://github.com/andela/ah-scorpion/blob/ch-update-README-instructions-159459817/README.md)

**What are the related Pivotal Tracker stories?**
[#159459817](https://www.pivotaltracker.com/story/show/159459817)

davenjeru committed:
[chore #159459817] update README.md instructions

Signed-off-by: davenjeru <dave.njeru@andela.com>

dmigwi commented:
You were supposed to append **Response** to https://github.com/andela/ah-scorpion#profile and all other responses from the various endpoints.

davenjeru committed:
[chore #159459817] update README.md instructions
 - add .test and .xml files to .gitignore. These files are automatically
   generated by Pycharm to show coverage on the editor

Signed-off-by: davenjeru <dave.njeru@andela.com>

davenjeru committed:
[chore #159459817] update README.md instructions
 - add the word 'Response' to the preferred response formats

Signed-off-by: davenjeru <dave.njeru@andela.com>
